src/main/java/io/harness/cf/client/Example.java

```diff
@@ -1,103 +1,69 @@
 package io.harness.cf.client;
 
-import io.harness.cf.client.api.CfClient;
+import com.google.gson.JsonObject;
+import io.harness.cf.client.api.Client;
+import io.harness.cf.client.api.Config;
+import io.harness.cf.client.api.FileMapStore;
 import io.harness.cf.client.dto.Target;
-import lombok.extern.slf4j.Slf4j;
-
 import java.util.HashMap;
-import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.Executor;
 import java.util.concurrent.Executors;
-import java.util.concurrent.TimeUnit;
+import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 class Example {
 
-    public static final int capacity;
-    public static final Executor executor;
-    public static final HashMap<String, String> keys;
-
-    public static final String FREEMIUM_API_KEY = "dac123e6-8a08-4775-825b-ed92583b5a70";
-    public static final String NON_FREEMIUM_API_KEY = "1acfded6-65b9-4e0a-9cbd-a6abd7574f54";
-
-    static {
-
-        capacity = 5;
-        keys = new HashMap<>(capacity);
-        keys.put("Freemium", FREEMIUM_API_KEY);
-        keys.put("Non-Freemium", NON_FREEMIUM_API_KEY);
-        executor = Executors.newSingleThreadExecutor();
+  private static final int capacity;
+  private static final Executor executor;
+  private static final HashMap<String, String> keys;
```

Comment:
you could move this static initializer into the Fields block, also using guava ImmutableHashMap. i.e. 
```suggestion
  private static final int capacity = 5;
  private static final Executor executor = Executors.newScheduledThreadPool(2);
  private static final HashMap<String, String> keys = ImmutableHashMap.of("Freemium", FREEMIUM_API_KEY, "Non-Freemium", NON_FREEMIUM_API_KEY);
```